playwright-e2e/dsm/component/tables/onc-history-table.ts

```diff
@@ -259,10 +260,12 @@ export default class OncHistoryTable extends Table {
     return this.row(index).locator('td').first();
   }
 
-  public row(index?: number): Locator {
-    return index
-      ? this.page.locator(this.tableXPath + this.rowXPath).nth(index)
-      : this.page.locator(this.tableXPath + this.rowXPath);
```

Comment:
Bug in this code. `index = 0` was evaluated to false.
It causes `multi-tabs.spec.ts` to be flaky.

```
multi-tabs.spec.ts:18:5 Multiple browser tabs @dsm
[dsm] › tests/dsm/multi-tabs.spec.ts:18:5 › Multiple browser tabs @dsm › Add Onc history for study PanCan 

    Error: Delete row button is not visible

    Timed out 30000ms waiting for expect(locator).toBeVisible()

    Locator: locator('//app-onc-history-detail//table[contains(@class,\'table\')]/tbody/tr').locator('td').last().getByRole('button')
    Expected: visible
    Received: hidden
    Call log:
      - Delete row button is not visible with timeout 30000ms
      - waiting for locator('//app-onc-history-detail//table[contains(@class,\'table\')]/tbody/tr').locator('td').last().getByRole('button')

```